### DISCUSSION TITLE
A great Concern: URL only images

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#10579] [Help] [answered]
Hi

I am on self hosted version on a LAN on Dockers(windows 10)
But the app logo needs Live URL.

What if internet service was down 
How to add Logo in the app in a LAN only setup? 
How to add more images? 
Is there a gallery option somewhere within budibase to select from there instead of URL from internet?

Why there is no local copy saved for Logo or other images?

**Comment by melohagan** [comment, marked as answer]
Hey @sghriyadh 

If you use the attachment field within an Internal Budibase DB, then the images will be uploaded to a local MinIO instance. So even though in this case it is still using a URL, it is pointing to a file within the docker container itself.

- https://docs.budibase.com/docs/image
- https://docs.budibase.com/docs/accessing-minio
